examples/tutorials/advanced/legends.py

```diff
@@ -0,0 +1,142 @@
+"""
+Creating legends
+================
+
+The :meth:`pygmt.Figure.legend` method creates legends, whereby auto-legends as
+well as manually created legends are supported.
+"""
+
+# %%
+import io
+
+import pygmt
+
+# %%
+# Create an auto-legend
+# ---------------------
+#
+# For auto-legends, the ``label`` parameter of :meth:`pygmt.Figure.plot` has to
+# be specified to state the desired text in the legend entry (white spaces are
+# allowed). Optionally, to adjust the legend, users can append different modifiers
+# to the string passed to ``label`. A list of all available modifiers can be
```

Comment:
```suggestion
# to the string passed to ``label``. A list of all available modifiers can be
```

Reply:
Several typo fixes are included in commit https://github.com/GenericMappingTools/pygmt/pull/3594/commits/f806a06e5ed0f57d743bdaa12781104a137eacf6.